server/channels/app/slashcommands/command_test.go

```diff
@@ -416,7 +422,8 @@ func TestDoCommandRequest(t *testing.T) {
 		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.Header().Add("Content-Type", "application/json")
 
-			io.Copy(w, InfiniteReader{})
+			_, err := io.Copy(w, InfiniteReader{})
+			require.NoError(t, err)
```

Comment:
Is the `InfiniteReader` implementation expected to return an error, perhaps from a `timeout` in the main function? If yes, then I think it should be `require.Error` and not `require.NoError` like I have it there. What do you think?

Reply:
`InfiniteReader` is just a mock implementation. One problem it has is that it doesn't return EOF when it's done. If you change that, it will fix this test case.

Reply:
Sorry, I don't understand the question. Why would increasing the reader response solve the EOF issue?

Reply:
I am hoping that since `InfiniteReader{}` simulates a reader that never ends wrapping it around `io.LimitedReader` will make it stop at the specified limit (1MB) hence returning an EOF.

Reply:
@hanzei, is there something else I am yet to do?

Reply:
@hanzei, what do you say, please?

Reply:
@hanzei, what do you say, please?